kinto/core/resource/__init__.py

```diff
@@ -4,6 +4,8 @@
 import colander
 import venusian
 import six
+from jsonpatch import JsonPatchException
+from jsonpointer import JsonPointerException
```

Comment:
I'd rather catch those to `utils.py` and raise a simple ValueError from them. From the resource point of view, these are details of implementation